src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -421,9 +414,9 @@ public static int execute(final String[] args) throws Exception {
         }
         
         
-        if(port < 9300) {
+        if(port < 9200) {
             System.out.println("WARNING: Seems you want connect to the OpenSearch HTTP port."+System.lineSeparator()
-                             + "         securityadmin connects on the transport port which is normally 9300.");
+                             + "         securityadmin connects on the transport port which is normally 9200.");
```

Comment:
Is this line#417 - line#419 still needed?
`transport port which is normally 9200` - This does not make sense to me.

Reply:
Yes, you are right, I did not get fully the initial message